sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -4061,6 +4079,26 @@ export enum KnownRecurrenceFrequency {
     Year = "Year"
 }
 
+// @public
+export enum KnownRequestStatus {
+    // (undocumented)
+    Completed = "Completed",
+    // (undocumented)
+    Failed = "Failed",
+    // (undocumented)
+    Running = "Running"
+}
+
+// @public
+export enum KnownResourceStatus {
```

Comment:
I wonder why did we switch from a union of string literals to enums?

Reply:
For extansible-enums these are for convenience as they are modeled as string because there is no way to represent in a meaningful way `type foo = "foo" | "bar" | "baz" | string` as it collapses to `string`

Reply:
Yeah, I meant why the known one is not defined as a union of literals without including the string type?